cmd/powbench/runner/runner.go

```diff
@@ -20,17 +19,17 @@ var (
 
 // TestSetup describes a test configuration.
 type TestSetup struct {
-	LotusAddr    multiaddr.Multiaddr
```

Comment:
Noticed this "lotus" naming, not sure what that meant. Seems like it should be "powergate".